tests/i18n/tests.py

```diff
@@ -533,6 +533,16 @@ def test_locale_independent(self):
             self.assertEqual('31.12.2009 в 20:50', Template('{{ dt|date:"d.m.Y в H:i" }}').render(self.ctxt))
             self.assertEqual('⌚ 10:15', Template('{{ t|time:"⌚ H:i" }}').render(self.ctxt))
 
+    def test_floatformat_force_grouping(self):
+        self.assertEqual('66,666.7', Template('{{ n|floatformat:"1g" }}').render(self.ctxt))
+        self.assertEqual('10,000', Template('{{ l|floatformat:"g" }}').render(self.ctxt))
+        # invalid suffix
+        self.assertEqual('10000', Template('{{ l|floatformat:"g2" }}').render(self.ctxt))
+
+        with translation.override('de'):
+            self.assertEqual('66.666,7', Template('{{ n|floatformat:"1g" }}').render(self.ctxt))
+            self.assertEqual('10.000', Template('{{ l|floatformat:"g" }}').render(self.ctxt))
```

Comment:
I moved this test to the `tests/template_tests/filter_tests/test_floatformat.py`.